src/Makefile

```diff
@@ -11,10 +11,14 @@ ifeq ($(OS),Windows_NT)
   FSC     = fsc
   MSBUILD = $(shell cygpath -u \"$$(reg.exe query 'HKLM\SOFTWARE\Microsoft\MSBuild\ToolsVersions\14.0' /v MSBuildToolsPath | tail -n 2 | head -n 1 | sed 's/.*REG_SZ\s\+//g')\")/MSBuild.exe
   UNAME   = Windows_NT
+  FSTAR   = ../bin/fstar.exe
+  RUNTIME =
```

Comment:
yes, I thought we could drop it, but we need it actually for the F# bootstrap. Though in all the other cases there is no ambiguity, because fslex, fsyacc, nuget, etc. are always mono assemblies and never native ones.